lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -206,9 +206,9 @@
                             title: gettext(platformData[1] + ' Link'),
                             valueAttribute: 'social_links',
                             helpMessage: gettext(
-                                'Enter your ' + platformData[1] + ' username or the URL to your ' +
-                                platformData[1] + ' page. Delete the URL to remove the link.'
-                            ),
+                                'Enter your ') + platformData[1] + gettext(' username or the URL to your ') +
```

Comment:
what is in platformData[1]?

Reply:
it is the list of social network platforms available .